.github/workflows/hitl.yml

```diff
@@ -16,3 +16,9 @@ jobs:
           event-type: stabilizer
           repository: quartiq/hitl
           client-payload: '{"github": ${{ toJson(github) }}}'
+
+      - uses: fountainhead/action-wait-for-check@v1.0.0
+        with:
+          token: ${{ secrets.GITHUB_TOKEN }}
+          checkName: HITL
```

Comment:
That's itself. That should block forever.

I think you want to rename [this one](https://github.com/quartiq/hitl/blob/main/.github/workflows/stabilizer.yml#L20), get it working and wait for it.

Reply:
https://github.com/quartiq/stabilizer/pull/332/checks?check_run_id=2304646861 seems to indicate it doesn't actually detect itself. In any case, I'm aware of this and planning on addressing it at a later step. This PR is still a draft as a means to force GHA and test the checks mechanism, so isn't ready for review at this point

Reply:
Also worth mentioning here after investigation: `HITL` isn't the name of this check - it's the name of the "check suite", so it doesn't actually register when querying for check results. The name of this check is actually the name of each job in this workflow, so it's likely some randomly assigned ID (we could specify it with `id`, but we don't care here).

This explains why it wasn't picking up itself earlier.